src/website/webview/templates/suggestion_list.html

```diff
@@ -24,11 +24,15 @@ <h1>Automatically generated suggestions</h1>
           {% severity_badge object.base_severity %}
         </div>
 
-        <h2>{{ object.package_name }}</h2>
-
         <details class="description">
           <summary>
-              {% if object.title %}{{ object.title }}{% else %}{{ object.description|truncatewords:10 }}{% endif %}
+            <h2>
+              {% if object.title %}
```

Comment:
Is the title really optional? I left the condition while refactoring because it was there originally, but I don't actually know the data quality here.